### PR TITLE
feat: add invite link URL support to team onboarding

### PR DESCRIPTION
**Pull request by CompuIves**
## What kind of change does this PR introduce?

Gives a link to invite team members

image

Tested it by clicking the button, and verifying that the right url was on clipboard :D

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7123/commits/9128432d05dc3173242706ccd0c2a47e2b032da2 is at https://pr7123.build.csb.dev/s/new.

File: packages/app/src/app/pages/common/Modals/NewTeamModal/TeamMembers.tsx [hunk 1]
```diff
@@ -35,10 +36,35 @@ export const TeamMembers: React.FC<{ onComplete: () => void }> = ({
...
+    }
+
+    actions.track({
+      name: 'Dashboard - Copied Team Invite URL',
+      data: { place: 'modal', inviteLink },
+    });
+
+    copyLinkTimeoutRef.current = window.setTimeout(() => {
+      setLinkCopied(false);
+    }, 1500);
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/common/Modals/NewTeamModal/TeamMembers.tsx:66]: Nitpick, but we can get rid of `window` for `*Timeout` functions if we want. We do have to type the timeout different in that case though.

```suggestion
  const copyLinkTimeoutRef = React.useRef<ReturnType<typeof setTimeout>>();

  const inviteLink = teamInviteLink(activeTeamInfo.inviteToken);

  const copyTeamInviteLink = () => {
    copyToClipboard(inviteLink);
    setLinkCopied(true);

    if (copyLinkTimeoutRef.current) {
      clearTimeout(copyLinkTimeoutRef.current);
    }

    actions.track({
      name: 'Dashboard - Copied Team Invite URL',
      data: { place: 'modal', inviteLink },
    });

    copyLinkTimeoutRef.current = setTimeout(() => {
      setLinkCopied(false);
    }, 1500);
```

**Review by tristandubbeld — APPROVED** [review]
Good to go :shipit:!

File: packages/app/src/app/pages/common/Modals/NewTeamModal/TeamMembers.tsx (hunk 1 above)
Comment (CompuIves) [reply packages/app/src/app/pages/common/Modals/NewTeamModal/TeamMembers.tsx:66]: Nice! Thanks for the tip!

**Pull request merged**